runtime.go

```diff
@@ -607,33 +579,23 @@ func getDockerVersion() (string, error) {
 
 }
 
-func getWorkingDir(basedir string, codeuri string, checkWorkingDirExist bool) (string, error) {
-
-	// Determine which directory to mount into the runtime container.
-	// If no CodeUri is specified for this function, then use the same
-	// directory as the SAM template (basedir), otherwise mount the
-	// directory specified in the CodeUri property.
-	abs, err := filepath.Abs(basedir)
-	if err != nil {
-		return "", err
-	}
-
-	dir := filepath.Join(abs, codeuri)
-
-	if checkWorkingDirExist {
+func getWorkingDir(dir string) string {
 
-		// ...but only if it actually exists
-		if _, err := os.Stat(dir); err != nil {
-			// It doesn't, so just use the directory of the SAM template
-			// which might have been passed as a relative directory
-			dir = abs
+	// If the template filepath isn't set, just use the cwd
+	if dir == "" {
+		cwd, err := os.Getwd()
```

Comment:
The CodeUri specified-as-a-file (e.g. unzipping JAR/ZIP archives) works, it just isn't handled in this method any more.

The previous iteration of this method was trying to do too many things. It had three potential inputs and 24 possible combinations to unit test (which I had written, but it was hard to make changes to).

I split out the CodeUri stuff to the unzipping section, and it makes for a much easier to understand method, that is easier to test.

This method *just* gets the working directory now, not working directory + maybe codeuri file too.